database/factories/ArtistFactory.php

```diff
@@ -2,10 +2,12 @@
 
 use Faker\Generator as Faker;
 
-$factory->define(App\Artist::class, function (Faker $faker) {
-    return [
+use App\Contracts\ArtistRepositoryInterface;
 
-    ];
+$repo = resolve(ArtistRepositoryInterface::class);
```

Comment:
should name this `$artist` as all dependencies will need to be resolved

Reply:
Yes indeed. Good call. Fixed in subsequent commit.